package.json

```diff
@@ -44,24 +44,22 @@
 	],
 	"scripts": {
 		"build": "node scripts/iu-mappings.js && node scripts/character-class-escape-sets.js",
-		"test": "mocha tests",
-		"cover": "istanbul cover --report html node_modules/.bin/_mocha tests -- -u exports -R spec"
+		"test": "node --test tests/tests.js",
+		"test-node6": "mocha tests",
+		"cover": "NODE_V8_COVERAGE=coverage node --test  --experimental-test-coverage tests/tests.js"
 	},
 	"dependencies": {
 		"regenerate": "^1.4.2",
-		"regenerate-unicode-properties": "^10.1.0",
-		"@babel/regjsgen": "^0.8.0",
-		"regjsparser": "^0.9.1",
+		"regenerate-unicode-properties": "^10.1.1",
+		"regjsgen": "^0.8.0",
+		"regjsparser": "^0.10.0",
 		"unicode-match-property-ecmascript": "^2.0.0",
 		"unicode-match-property-value-ecmascript": "^2.1.0"
 	},
 	"devDependencies": {
-		"codecov": "^3.8.3",
```

Comment:
It seems that we are not using `codecov` anyway. If we want to use it we can add a GH action `codecov-action` later.

Reply:
It would be great to add it 👍